base/setup/reactos/lang/fr-FR.rc

```diff
@@ -258,33 +258,33 @@ Les caractères d'espacement ne sont pas autorisés."
+    IDS_ERROR_CREATE_PARTITION_TITLE "Créer une partition"
+    IDS_ERROR_CREATE_PARTITION "Echec lors de la création d'une nouvelle partition."
 
-    IDS_WARN_DELETE_PARTITION_TITLE "Delete partition?"
-    IDS_WARN_DELETE_PARTITION "Are you sure you want to delete the selected partition?"
-    IDS_WARN_DELETE_MBR_EXTENDED_PARTITION "Are you sure you want to delete the selected extended partition and ALL the logical partitions it contains?"
+    IDS_WARN_DELETE_PARTITION_TITLE "Détruire la partition ?"
+    IDS_WARN_DELETE_PARTITION "Êtes-vous sûr de vouloir détruire la partition sélectionnée ?"
+    IDS_WARN_DELETE_MBR_EXTENDED_PARTITION "Êtes-vous sûr de vouloir détruire la partition étendue sélectionnée et TOUTES les partitions logiques qu'elle contient ?"
 
-    IDS_ERROR_WRITE_PTABLE "Setup failed to write partition tables."
+    IDS_ERROR_WRITE_PTABLE "Le programme d'installation a échoué à écrire les tables de partition."
 
-    IDS_ERROR_SYSTEM_PARTITION "The ReactOS Setup could not find a supported system partition\n\
-on your system or could not create a new one. Without such a partition\n\
-the Setup program cannot install ReactOS.\
-\nClick on OK to return to the partition selection list."
+    IDS_ERROR_SYSTEM_PARTITION "Le programme d'installation ReactOS ne peut pas trouver un système de partition supporté\n\
+sur votre système ou ne peut pas en créer une nouvelle. Sans une telle partition\n\
+le programme d'installation ne peut pas installer ReactOS.\
+\nCliquez sur OK pour retourner à la liste de sélection des partitions."
 
-    IDS_ERROR_FORMAT_UNRECOGNIZED_VOLUME "Unrecognized volume while attempting to format the partition."
+    IDS_ERROR_FORMAT_UNRECOGNIZED_VOLUME "Volume non reconnu lors du formattage de la partition."
 
-    IDS_ERROR_COULD_NOT_FORMAT "Setup is currently unable to format a partition in %s.\n\
-\nClick on OK to continue Setup.\
-\nClick on CANCEL to quit Setup."
+    IDS_ERROR_COULD_NOT_FORMAT "Le programme d'installation ne parvient actuellement pas à formatter une partition dans %s.\n\
+\nCliquez sur OK pour continuer l'installation.\
+\nCliquez sur CANCEL pour quitter l'installation."
 
-    IDS_ERROR_FORMATTING_PARTITION "Setup is unable to format the partition:\n %s\n"
+    IDS_ERROR_FORMATTING_PARTITION "Le programme d'installation ne parvient pas à formatter la partition :\n %s\n"
 
-    IDS_ERROR_COULD_NOT_CHECK "Setup is currently unable to check a partition formatted in %s.\n\
-\nClick on OK to continue Setup.\
-\nClick on CANCEL to quit Setup."
+    IDS_ERROR_COULD_NOT_CHECK "Le programme d'installation ne parvient actuellement pas à vérifier une partition formattée dans %s.\n\
+\nCliquez sur OK pour continuer l'installation.\
+\nCliquez sur CANCEL pour quitter l'installation."
 
-    IDS_ERROR_CHECKING_PARTITION "ChkDsk detected some disk errors.\n(Status 0x%08lx)."
+    IDS_ERROR_CHECKING_PARTITION "ChkDsk a détecté des erreurs disque.\n(Status 0x%08lx)."
```

Comment:
Please move the translations made in any modules inside base/setup (including this base/setup/reactos/lang/fr-FR.rc) to a separate PR, since the GUI setup is actively being developed.